### PR TITLE
added v1 jar signature requirement

### PR DESCRIPTION
**Pull request by svedule**

### CODE REVIEW

File: _posts/2011-01-01-how-to-submit-apk.md [hunk 1]
```diff
@@ -15,6 +15,35 @@ date: 2013-01-06 07:05:00
 }
 </style>
 
+<br>
+## Pre-Requisites
+
+Before uploading to Poynt developer portal, APKs must be digitally signed with android keys. Steps as follows:
```
Comment (dsnatochy) [inline _posts/2011-01-01-how-to-submit-apk.md]: Let's rephrase:
_You need to sign your apk before uploading it in the developer portal:_

File: _posts/2011-01-01-how-to-submit-apk.md [hunk 2]
```diff
@@ -15,6 +15,35 @@ date: 2013-01-06 07:05:00
...
+
+### a. Generate an Upload key and keystore
+
+In Android Studio select `Build->Generate Signed Bundle/APK`
+
+<img class="shadow image" src="../assets/app-signing-1.png" alt="app-signing-1" style="border:20px;margin:20px">
```
Comment (dsnatochy) [inline _posts/2011-01-01-how-to-submit-apk.md]: Can you reduce the width of the first 3 screenshots by 2x?

**Review by dsnatochy — CHANGES_REQUESTED** [review]
left a few comments

**Comment by svedule** [comment]
Pushed changes.

**Review by dsnatochy — APPROVED** [review]

**Pull request merged**